src/core/types/decimal192.rs

```diff
@@ -351,8 +368,8 @@ pub fn new_decimal_from_i64(value: i64) -> Decimal192 {
 /// so by first converting the float to a String, using
 /// Rust's `to_string` on the float.
 #[uniffi::export]
-pub fn new_decimal_from_f32(value: f32) -> Decimal192 {
-    value.into()
+pub fn new_decimal_from_f32(value: f32) -> Result<Decimal192, CommonError> {
```

Comment:
`Result<Decimal192, CommonError>` can be written just `Result<Decimal192>`

Reply:
you can copy paste the documentation of the `TryFrom` impl above (and change doc test to use `new_decimal_from_f32`) perhaps?